lib/matplotlib/tests/test_mlab.py

```diff
@@ -80,33 +80,14 @@ def test_stride_windows_n32_noverlap0_unflatten(self, axis):
         assert_array_equal(y, x1)
 
 
-class TestWindow:
-    def setup(self):
-        np.random.seed(0)
-        n = 1000
-
-        self.sig_rand = np.random.standard_normal(n) + 100.
-        self.sig_ones = np.ones(n)
-
-    def test_window_none_rand(self):
-        res = mlab.window_none(self.sig_ones)
-        assert_array_equal(res, self.sig_ones)
-
-    def test_window_none_ones(self):
-        res = mlab.window_none(self.sig_rand)
-        assert_array_equal(res, self.sig_rand)
-
-    def test_window_hanning_rand(self):
-        targ = np.hanning(len(self.sig_rand)) * self.sig_rand
-        res = mlab.window_hanning(self.sig_rand)
-
-        assert_allclose(targ, res, atol=1e-06)
-
-    def test_window_hanning_ones(self):
-        targ = np.hanning(len(self.sig_ones))
-        res = mlab.window_hanning(self.sig_ones)
-
-        assert_allclose(targ, res, atol=1e-06)
+def test_window():
+    n = 1000
```

Comment:
Why are we dropping the call to `np.random.seed(0)`?

Reply:
I accidentally dropped it, but again if you looked at the content of the tests it's clear that this should *always* pass regardless of the seed.

Reply:
True, but in the unlikely even that it ever should fail, it's completely unreproducible with the seed unset.